ui/apps/platform/src/types/table.ts

```diff
@@ -0,0 +1,17 @@
+import { ReactElement } from 'react';
+
+export type { ThProps } from '@patternfly/react-table';
```

Comment:
The re-export here was solely to remove the PF dependency from `useURLSort` while still being able to reuse the existing type.